js/okex.js

```diff
@@ -1499,6 +1499,14 @@ module.exports = class okex extends Exchange {
         return this.parseTradingFee (first, market);
     }
 
+    async parseBalance (response, params) {
+        const defaultType = this.safeString (this.options, 'defaultType');
```

Comment:
I'm no sure if it's necessary to duplicate these lines, also, by the time we reach this place in code, the `type` has been already omitted from `params` in line 1516, so it simply won't work this way. We might have to pass the type to it.